src/node.rs

```diff
@@ -244,17 +244,11 @@ impl Element {
 
         for a in attributes {
             let name_local = a.name.local.deref();
-            let mut value = "";
 
             if name_local == "id" {
-                value = a.value.deref();
-                id = Some(LocalName::from(value));
-            }
-            if name_local == "class" {
-                if value.is_empty() {
-                    value = a.value.deref();
-                }
-                classes.extend(value.split_whitespace().map(LocalName::from));
+                id = Some(LocalName::from(a.value.deref()));
+            } else if name_local == "class" {
```

Comment:
was rushing and made this change without thinking. @adamreichold @cfvescovo  feel free to squash the commits for this PR.

Reply:
Will do, thanks